incubator/patroni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: patroni
 description: "Highly available elephant herd: HA PostgreSQL cluster."
-version: 0.2.0
+version: 0.2.1
```

Comment:
For the Bitnami images we use tag with the format `<upstream version>-r<image version>`, e.g. `1.2.3-r1`. For these, I've just been using the upstream version as the appVersion.